client/src/Clusters/TopCharts/TopCharts.js

```diff
@@ -0,0 +1,191 @@
+import React, { Component } from "react";
+import "./SingleAppContainer.js";
+import "./TopChartsStyle.css";
+import "../styles.css";
+import SingleAppContainer from "./SingleAppContainer.js";
+ 
+class TopCharts extends Component {
+  render() {
+    const makeColumn = item => (
+      <div>
+        <SingleAppContainer
+          listNum={item.listNum}
+          url={item.url}
+          title={item.title}
+          category={item.category}
+          rating={item.rating}
+        />
+      </div>
+    );
+    console.log("inside");
+    const emptyAppImage =
+      "data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAANwAAADlCAMAAAAP8WnWAAAAElBMVEWytbXQ0tLLzc26vb2vsrLT1dVz9wdHAAABBklEQVR4nO3PARGAMAwAsQ6Gf8voSO/jIPPcWeqeKUcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqpyqnKqcqp9qeO+9a3w8XGAxhjK6uyAAAAABJRU5ErkJggg==";
+ 
+    const topCharts = [
+      {
+        listNum: 1,
```

Comment:
Each cluster should also have a unique ID. you can either hardcode or use https://www.npmjs.com/package/uuid to generate